plugins/sample/controllers/index.js

```diff
@@ -0,0 +1,273 @@
+/*
```

Comment:
This entire file is just for testing. It will be removed before we merge. This file uses localization and ls.g instead of ls.get
If youd like to test with this file go to /about after you've installed the sample plugin. 
To test site overrides with this file go to the /admin/localization route and add a locale called SAMPLE_HELLO_WORLD with whatever text you'd like then visit the /about page to see it update with that text.
NOTE: only files using ls.g will get the benefits of this PR.